dataactvalidator/config/supervisord.conf

```diff
@@ -20,6 +21,8 @@ directory=/data-act/backend/dataactvalidator/
 startsecs=1
 startretries=10
 user=root
+environment=PYTHONPATH=%(ENV_PATH)s:/data-act/backend
```

Comment:
Ooooh, this is Python templating, not sh templating! Never mind my comment, then!